setup.py

```diff
@@ -50,6 +58,7 @@
         "Topic :: Scientific/Engineering :: Physics",
         "Development Status :: 5 - Production/Stable",
     ],
+    setup_requires=["numpy>=1.16.0"],
```

Comment:
Can we change this to package named "oldest-supported-numpy" ?
That one will automatically pick oldest supported Numpy for corresponding Python version.
See [here](https://github.com/nikolasibalic/ARC-Alkali-Rydberg-Calculator/issues/100#issuecomment-1036258981) for the reasons.
`setup_requires=["oldest-supported-numpy"],`